ios/RN/RNCamera.m

```diff
@@ -1058,7 +1058,7 @@ - (void)takePicture:(NSDictionary *)options resolve:(RCTPromiseResolveBlock)reso
                             path = options[@"path"];
                         }
                         else{
-                            path = [RNFileSystem generatePathInDirectory:[[RNFileSystem cacheDirectoryPath] stringByAppendingPathComponent:@"Camera"] withExtension:@".jpg"];
+                            path = [self getImagePath];
```

Comment:
can we just specify path in the options? see line 1058

Reply:
nit: checked the above and currently options path is not supported for heic. We can add it in to be controlled from the js thread